backend/tests/api-e2e/test_api_delegation.py

```diff
@@ -59,5 +59,49 @@ def test_delegation(client: Client, payload: ScoreDelegationPayload):
 
 
 @pytest.mark.api
-def test_check_in_delegation():
-    pass
+def test_recalculate_in_delegation(client: Client, payload: ScoreDelegationPayload):
+    """
+    Recalculation can actually return two different results:
+    - if the delegation does not exist, it will return 400
+    - if the delegation exists, i.e. secondary address exists in the database, it will return 400
+    it's due to the fact that the recalculation is already stoned for a secondary address in our implementation
```

Comment:
> score is already set in stone

Reply:
Or possibly - petrified.

Reply:
added an attempt of doing a redelegation with the same secondary address and checking if it returns a correct response